lib/xcode/install.rb

```diff
@@ -141,21 +141,28 @@ def current_symlink
       File.symlink?(SYMLINK_PATH) ? SYMLINK_PATH : nil
     end
 
-    def download(version, progress, url = nil, progress_block = nil, retry_download_count = 3)
+    def download(version, progress, url = nil, local_url = nil, progress_block = nil, retry_download_count = 3)
       xcode = find_xcode_version(version) if url.nil?
       return if url.nil? && xcode.nil?
 
       dmg_file = Pathname.new(File.basename(url || xcode.path))
 
-      result = Curl.new.fetch(
-        url: url || xcode.url,
-        directory: CACHE_DIR,
-        cookies: url ? nil : spaceship.cookie,
-        output: dmg_file,
-        progress: progress,
-        progress_block: progress_block,
-        retry_download_count: retry_download_count
-      )
+      result = false
```

Comment:
Instead of using a boolean to track this, how about a simple if/else statement? 

```ruby
if local_url.nil?
    # Existing code
else
    # Your new code
end
```